api/openapi.yaml

```diff
@@ -160,6 +162,9 @@ components:
         httpsForward: true
         contentType: true
         reachable: true
+        checkedVersions:
+        - checkedVersions
+        - checkedVersions
```

Comment:
What does this mean?

Reply:
Nothing really. This is a (in my opinion) rather stupid example that the OpenAPI generator created. My guess is that since the `checkedVersions` field is defined to hold a list of arbitrary strings, the generator just picked two times the string `checkedVersions` for the generated example.